calibrate.py

```diff
@@ -352,12 +351,35 @@ def __init__(self):
                 self.board_config = json.load(fp)
                 self.board_config = self.board_config['board_config']
                 self.board_config_backup = self.board_config
+        else:
+            cameraProperties = self.device.getConnectedCameraFeatures()
+            calibData = self.device.readCalibration()
+            eeprom = calibData.getEepromData()
+            print(f"Product name: {eeprom.productName}, board name {eeprom.boardName}")
+            detection = eeprom.productName.split()
+            if "AF" in detection:
+                detection.remove("AF")
+            if "FF" in detection:
+                detection.remove("FF")
+            if "9782" in detection:
+                detection.remove("9782")
+            self.board_name = '-'.join(detection).upper()
+            board_path = Path(self.board_name)
+            if not board_path.exists():
+                board_path = (Path(__file__).parent / 'resources/depthai_boards/boards' / self.board_name.upper()).with_suffix('.json').resolve()
+                if not board_path.exists():
+                    raise ValueError(
+                        'Board config not found: {}'.format(board_path))
+            with open(board_path) as fp:
+                self.board_config = json.load(fp)
+                self.board_config = self.board_config['board_config']
+                self.board_config_backup = self.board_config
```

Comment:
```device.getDeviceName``` instead of board_name & upper

and cameraProperties & hasAutofocus instead of parsing AF FF etc...